portfolio/src/main/webapp/contact.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<meta charset="UTF-8">
+<title>Roland's - Portfolio</title>
+<link rel="stylesheet" href="style.css">
+<script src="script.js"></script>
+
+<nav id="navbar">
+	<div id="nav-left-side">
+		<a href="/index.html" class="nav-button navbar-item">NR</a>
+		<a href="/index.html" class="nav-button navbar-item">Home</a>
+		<a href="/works.html" class="nav-button navbar-item">Work</a>
+		<a href="/contact.html" class="nav-button navbar-item">Contact</a>
+	</div>
+	<div id="nav-right-side">
+		<a href="https://www.linkedin.com/in/roland-naijuka" target="_blank" class="nav-button navbar-item">LinkedIn</a>
+		<a href="mailto:naijuka.roland@outlook.com" target="_blank" class="nav-button navbar-item">Email</a>
+	</div>
+</nav>
+
+<div id="contact-me">
+	<h1>Contact Me!</h1>
+	<form action="#" method="POST">
```

Comment:
Are you handling the POST somewhere?  If not, you might want to remove that attribute